### PR TITLE
fix export-pdfa notebook

### PR DESCRIPTION
**Pull request by felixdittrich92**
- remove `.as_images()`
- add missing spaces while overlaying

### CODE REVIEW

**Comment by felixdittrich92** [comment]
@frgfm

File: doctr/export_as_pdfa.ipynb [hunk 1]
```diff
@@ -13,7 +13,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 9,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: That should be the first cell (or null)  (cells are supposed to be executed in the correct order, so at worst, it should the first cell to be executed

File: doctr/export_as_pdfa.ipynb [hunk 2]
```diff
@@ -79,7 +93,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 57,
+      "execution_count": 10,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: same

File: doctr/export_as_pdfa.ipynb [hunk 3]
```diff
@@ -120,7 +134,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 58,
+      "execution_count": 11,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: same

File: doctr/export_as_pdfa.ipynb [hunk 4]
```diff
@@ -277,7 +296,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 59,
+      "execution_count": 12,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: same

File: doctr/export_as_pdfa.ipynb [hunk 5]
```diff
@@ -357,7 +376,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 60,
+      "execution_count": 13,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: same

File: doctr/export_as_pdfa.ipynb [hunk 6]
```diff
@@ -386,7 +405,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 61,
+      "execution_count": 14,
```
Comment (frgfm) [inline doctr/export_as_pdfa.ipynb]: same

**Review by frgfm — COMMENTED** [review]
Thanks! Slight comment about cell execution order

**Review by frgfm — APPROVED** [review]
Looks good, thanks 🙏

**Pull request merged**